modules/python-modules/syslogng/modules/kubernetes/scl/kubernetes.conf

```
@@ -136,6 +137,6 @@ block source kubernetes(prefix('.k8s.') base-dir("/var/log/containers") cluster-
                 flags(no-parse)
             );
         };
-        parser { kubernetes-file-parser(prefix(`prefix`) cluster-name(`cluster-name`)); };
+        parser { kubernetes-file-parser(prefix(`prefix`) cluster-name(`cluster-name`) key-delimiter(`key-delimiter`); };
```

A closing bracket is missing.
```suggestion
        parser { kubernetes-file-parser(prefix(`prefix`) cluster-name(`cluster-name`) key-delimiter(`key-delimiter`)); };
```